tests/product/standalone/test_installation.py

```diff
@@ -55,10 +55,17 @@ def test_install_non_root(self):
         """.format(mount_dir=self.cluster.mount_dir,
                    install_dir=install_dir)
 
-        self.assertRaisesRegexp(OSError, 'mkdir: cannot create directory '
-                                '`/var/log/prestoadmin\': Permission denied',
-                                self.cluster.run_script_on_host, script,
-                                self.cluster.master)
+        self.cluster.run_script_on_host(script, self.cluster.master)
```

Comment:
So if we're still keeping this test, how are the remaining tests being run? As root? I saw that you set `invoke_sudo` to False. Can you clarify?